packages/components/src/presentation-components/instance-filter-builder/MultiTagSelect.tsx

```diff
@@ -29,6 +29,8 @@ import { AsyncPaginate, AsyncPaginateProps } from "react-select-async-paginate";
 import { SvgCaretDown, SvgCaretDownSmall, SvgCheckmarkSmall, SvgCloseSmall } from "@itwin/itwinui-icons-react";
 import { useResizeObserver } from "../common/Utils";
 
+export const EMPTY_PROPERTY_TEXT = "Empty Value";
```

Comment:
This doesn't localize the value. Please have a look at [this PR](https://github.com/iTwin/presentation/pull/176), specifically search for `unique-values-property-editor` usage